index.html

```diff
@@ -356,7 +356,7 @@ <h2><span class="glyphicon glyphicon-envelope" aria-hidden="true"></span> Connec
         <tbody>
           <tr>
             <td>Server</td>
-            <td>irc.wwivbbs.org:6697 (use SSL)</td>
```

Comment:
Did the CNAME trip up the SSL?

Reply:
IIRC, yes.  It was the mismatch between the cert (venomslair.com) and the domain (irc.venomslair.com)